NitroxServer/Serialization/ServerConfig.cs

```diff
@@ -47,11 +47,11 @@ public int SaveInterval
             get
             {
                 int configValue;
-                if (_saveInterval == null && Int32.TryParse(ConfigurationManager.AppSettings[DEFAULT_SAVE_SETTING], out configValue))
+                if (_saveInterval == null && Int32.TryParse(Properties.ServerSessionSettings.Default.SaveInterval, out configValue))
```

Comment:
Use the `ConfigurationManager.AppSettings` instead of a new settings location/file.